src/components/navbar/navDropdown/NavDropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../utils/variables';
+
+.dropdownContainer {
+    z-index: 1;
```

Comment:
The `navContainer` `z-index` is 20 so won't it overlap this?

Reply:
It won't because it is absolute, but yeah, it is absolute. I will remove this, thanks.